content/en/docs/instrumentation/js/automatic/_index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Automatic Instrumentation
+linkTitle: Automatic
+weight: 30
+---
+
+Automatic instrumentation provides a way to instrument any Node.js application
+and capture telemetry data from many popular libraries and frameworks without
+any code changes.
+
+## Setup
+
+Run the following commands to install the appropriate packages.
+
+```shell
+npm install --save @opentelemetry/api
+npm install --save @opentelemetry/auto-instrumentations-node
+```
+
+The `@opentelemetry/api` and `@opentelemetry/auto-instrumentations-node`
+packages install the API, SDK, and the instrumentation tools.
+
+## Configuring the module
+
+The module is highly configurable.
+
+One option is to configure the module by way of using `env` to set environment
+variables from the CLI:
+
+```shell
+env OTEL_TRACES_EXPORTER=otlp OTEL_EXPORTER_OTLP_TRACES_ENDPOINT=your-endpoint \
+node --require @opentelemetry/auto-instrumentations-node/register app.js
+```
+
+Alternatively, you can use `export` to set environment variables:
+
+```shell
+export OTEL_TRACES_EXPORTER="otlp"
+export OTEL_EXPORTER_OTLP_ENDPOINT="your-endpoint"
```

Comment:
following up on that one, maybe also have `OTEL_METRICS_EXPORTER` defined (although both are otlp by default)

Reply:
Added. I tried including a list of what could be `commonly` used env vars as an example.
Let me know if there are any more vars that can be relevant to this example.